translations/strings.yaml

```diff
@@ -443,10 +443,81 @@ mobileOnboarding:
   alreadyASubscriber: Already a subscriber?
 
 inAppAuth:
+  emailInputPlaceholder: Enter email
   changeEmailLink: Change email
   signInButton: Sign in
   signInSubtitle: Continue to Mozilla VPN by signing in with your Firefox account
+  createPasswordLabel: Create password
   forgotPasswordLink: Forgot your password?
   passwordInputPlaceholder: Enter password
   passwordInputLabel: Password
-  invalidPasswordErrorMessage: Invalid password. Please try again
+  passwordHintCharacterLength: 
+    value: Must be at least 8 characters
+    comment: This is a hint for the user - they must provide a password with at least 8 characters 
+  passwordHintEmailAddressAsPassword: 
+    value: Must not be your email address
+    comment: This is a hint for the user - they must provide a password that does not match their email. 
+  passwordHintCommonPassword: 
+    value: Must not be a common password
+    comment: This is a hint for the user - password they provide must not be a common password. Ex. password.
+  productUpdateOptInDescription: 
+    value: I’d like to receive product update emails from Firefox.
```

Comment:
Is this a checkbox? Do you have screenshots/mock-ups that I can look at?

Reply:
Yep, it is a checkbox label. 
From Figma:
 <img width="252" alt="Screen Shot 2022-03-14 at 8 50 44 AM" src="https://user-images.githubusercontent.com/22355127/158185791-72438c81-3ff3-4198-8632-8504bcb7f98e.png">